sdks/android/Mentat/library/src/main/java/org/mozilla/mentat/Mentat.java

```diff
@@ -36,7 +36,7 @@ public Mentat(String dbPath) {
     }
 
     /**
-     * Open a connection to an in-memory Store.
+     * Open a connection to an anonymous in-memory Store.
```

Comment:
This is very misleading, and is really the _problem_ here -- your store isn't anonymous, it's the "empty string", hence it always open a _new_ store.  Why not make this function actually anonymous by randomizing the string and drop all the hand-coded names?